build.go

```diff
@@ -884,8 +884,20 @@ func shouldRebuildAssets(target, srcdir string) bool {
 }
 
 func updateDependencies() {
-	runPrint(goCmd, "get", "-u", "./cmd/...")
-	runPrint(goCmd, "mod", "tidy", "-go=1.17", "-compat=1.17")
+	// Figure out desired Go version
+	bs, err := os.ReadFile("go.mod")
+	if err != nil {
+		log.Fatal(err)
+	}
+	re := regexp.MustCompile(`(?m)^go\s+([0-9.]+)`)
+	matches := re.FindStringSubmatch(string(bs))
```

Comment:
Actuall I beliebe it should work with `FindSubmatch()` and not converting to string?

Reply:
More or less... the result needs to be stringified. But this shaves of a couple of allocations probably, sure. :)